docs/development/affiliated-packages.rst

```diff
@@ -404,9 +404,9 @@ files manually`_ section since this explains what many of the files do.
    replace ``packagename`` with the name of your package.
 
 #. If you want the documentation for your project to be hosted by
-   `ReadTheDocs <https://readthedocs.org>`_, then you need to setup an
+   `ReadTheDocs <https://readthedocs.io>`_, then you need to setup an
```

Comment:
Thanks for pointing that out! It must have anticipated people like me... :smile: